resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreaker.java

```diff
@@ -298,12 +329,13 @@ public String toString(){
 
         EventPublisher onStateTransition(EventConsumer<CircuitBreakerOnStateTransitionEvent> eventConsumer);
 
-        EventPublisher onStateReset(EventConsumer<CircuitBreakerOnResetEvent> eventConsumer);
+        EventPublisher onReset(EventConsumer<CircuitBreakerOnResetEvent> eventConsumer);
 
         EventPublisher onIgnoredError(EventConsumer<CircuitBreakerOnIgnoredErrorEvent> eventConsumer);
 
         EventPublisher onCallNotPermitted(EventConsumer<CircuitBreakerOnCallNotPermittedEvent> eventConsumer);
 
+        EventPublisher onAny(EventConsumer<CircuitBreakerEvent> onCallNotPermittedEventConsumer);
```

Comment:
`onAny` is not required. There is already an `onEvent` in the `EventPublisher` interface.